### ISSUE TITLE
Eth Supply Projection - Amount of Eth "in contracts" is wrong

### ISSUE THREAD

**Issue by ckoopmann** [ultrasoundmoney/frontend#297]
The share of Eth "in contracts" as displayed in the `Eth Supply Projection 2Y` chart is wrong / stuck.
![image](https://github.com/ultrasoundmoney/frontend/assets/15629702/d678cf4f-7754-402d-88e6-5e2e6b368a18)

**Comment by JustinDrake** [comment]
Right—I believe we used https://studio.glassnode.com/metrics?a=ETH&category=&m=distribution.SupplyContracts and need to pay the annual subscription to get API access.

**Comment by JustinDrake** [comment]
The data also shows in this widget under "defi (stale)". Fixing this would make the widget much more screenshotable :)

<img width="525" alt="Screenshot 2024-01-16 at 11 18 02" src="https://github.com/ultrasoundmoney/frontend/assets/731743/b7aa067c-125a-4edf-b018-8b45f5ce347f">
